modules/channel-teams/README.md

```diff
@@ -0,0 +1,27 @@
+Follow these instruction to use the Botpress Teams channel:
+
+1. First read the official: [Microsoft bot framework documentation](https://docs.microsoft.com/en-us/microsoftteams/platform/concepts/bots/bots-create)
+
+2. Try deploying an echo bot from the official [Microsoft bot framework samples](https://github.com/microsoft/BotBuilder-Samples). Don't dig to far in the sdk as Botpress handle this part of the logic.
+
+Once that you are familiar with Azure, that you've registered a Microsoft App Id and a password (sometimes called secret) and that you've achieved interacting with your deployed bot on MS Teams:
+
+3. Enable the channel-teams module in:
+
+   > data/global/botpress.config.json
+
+4. Create the following file:
+
+   > data/bots/YOUR_BOT_ID/config/channel-teams.json
+
+5. Write the following content in the file:
+
+   ```
+   {
+     "enabled": true,
+     "microsoftAppId": "YOUR APP ID",
+     "microsoftAppPassword": "YOUR APP PW"
+   }
+   ```
+
+You should be able to use your bot just like the sample bot you created and deployed at step 2.
```

Comment:
Can you provide a step-by step guide on how to do that? The user shouldn't have to go through the official documentation or the github repo to do all this stuff. Why the echo dot deployment? It shouldn't be necessary if instructions are clear enough

The user should be able to download the binary and create his bot by following a simple guide, without spending time reading docs of other providers (check the channel-slack readme for an example)

Reply:
That's the problem... There can't be a simple guide because deploying on MS Teams using MS botframework was kind of a nightmare. There's many steps and many ways to achieve the same thing. 

If a user wants to use the Teams channel, he needs first to know what are the appId and appPassword. These are the only informations we need.

I prefer refering the user to the actual MS documentation which will be more complete and will be updated by the microsoft team itself.